packages/core/src/JWEditor.ts

```diff
@@ -21,10 +23,11 @@ export class JWEditor {
     dispatcher: Dispatcher;
     eventManager: EventManager;
     pluginsRegistry: JWPlugin[];
-    renderer: Renderer;
+    renderers: Record<string, Renderer> = {};
```

Comment:
I suggest to give a nickname to that `string`. Maybe `RenderingOutputType` ? That's a bit long, though.